src/Item/Item.js

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from "react";
+import "./Item.css";
+
+const Item = ({ key, id, item }) => {
+  const [isNeeded, setIsNeeded] = useState(false);
+  return (
+    <article className="Item">
+      <img
+        id="check-or-uncheck"
+        alt=""
+        src={
+          !isNeeded
+            ? require("../assets/unchecked.svg")
```

Comment:
I think maybe we should import these at the top and just use the variable here?

Reply:
Good idea!